homeassistant/components/qbittorrent/coordinator.py

```diff
@@ -31,8 +44,21 @@ def __init__(self, hass: HomeAssistant, client: Client) -> None:
             update_interval=timedelta(seconds=30),
         )
 
-    async def _async_update_data(self) -> dict[str, Any]:
+    def update(self) -> SessionStats:
+        """Get the latest data from Transmission instance."""
```

Comment:
This is toooo in line with Transmission :P

Reply:
😂 I thought I caught all those. Nice find! Will fix this